backend/src/main/java/net/finman/dao/InvoiceDao.java

```diff
@@ -25,13 +26,13 @@
      * @param items        Items to be added to an invoice.
      * @throws ResourceNotCreatedException If the sql insertion failed.
      */
-    void addInvoiceItems(UUID serialNumber, String seller, List<Item> items) throws ResourceNotCreatedException;
+    void addInvoiceItems(UUID serialNumber, String seller, List<InvoiceItem> items) throws ResourceNotCreatedException;
```

Comment:
We don't need this function addInvoiceItems(...) anymore since invoiceItems are added through an invoice directly